apache/ambari
# AMBARI-24427: Fix broken Python UTs in ambari-server code -- Part 2

## What changes were proposed in this pull request?

Fix all broken python UT cases in ambari-server

## How was this patch tested?

 Manually tested:

----------------------------------------------------------------------
Ran 293 tests in 7.569s

OK
----------------------------------------------------------------------
Total run:323
Total errors:0
Total failures:0
OK
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle) @ ambari-server ---
[INFO] Starting audit...
Audit done.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:40 min
[INFO] Finished at: 2018-08-19T06:35:35Z
[INFO] Final Memory: 93M/594M
[INFO] ------------------------------------------------------------------------
